Code/Editor/TrackView/TVSequenceProps.cpp

```diff
@@ -74,12 +74,21 @@ bool CTVSequenceProps::OnInitDialog()
     Range timeRange = m_pSequence->GetTimeRange();
     float invFPS = 1.0f / m_FPS;
 
-    m_timeUnit = Seconds;
     ui->START_TIME->setValue(timeRange.start);
     ui->START_TIME->setSingleStep(invFPS);
     ui->END_TIME->setValue(timeRange.end);
     ui->END_TIME->setSingleStep(invFPS);
 
+    if (m_pSequence->GetFlags() & IAnimSequence::eSeqFlags_DisplayAsFramesOrSeconds)
```

Comment:
uber nit: It's generally recommended to prefer the explicit boolean check of `(m_pSequence->GetFlags() & IAnimSequence::eSeqFlags_DisplayAsFramesOrSeconds) != 0` (as opposed to the implicit conversion to `bool` from `int`) but in most cases it isn't a huge problem.